truss/_integration-tests/cli/cli_test.go

```diff
@@ -342,16 +343,23 @@ func cleanTests(servicesDir string) {
 		if !d.IsDir() {
 			continue
 		}
-		removeTestFiles(servicesDir + "/" + d.Name())
+		removeTestFiles(filepath.Join(servicesDir, d.Name()))
 	}
 }
 
 // removeTestFiles removes all files created by running truss and building the
 // service from a single definition directory
 func removeTestFiles(defDir string) {
-	if fileExists(defDir + "/TEST-service") {
-		os.RemoveAll(defDir + "/TEST-service")
-		os.RemoveAll(defDir + "/third_party")
-		os.RemoveAll(defDir + "/bin")
-	}
+	os.RemoveAll(filepath.Join(defDir, "TEST-service"))
+	os.RemoveAll(filepath.Join(defDir, "bin"))
+	os.RemoveAll(filepath.Join(defDir, "pbout"))
+	mkdir(filepath.Join(defDir, "pbout"))
+}
+
+// mkdir acts like $ mkdir -p path
+func mkdir(path string) error {
+	// 0775 is the file mode that $ mkdir uses when creating a directoru
+	err := os.MkdirAll(path, 0775)
```

Comment:
I've never seen 0775 as a default, personally. 0755 for directories and executables, and 0644 for regular files, is what I've seen nearly everywhere. What you get depends on your umask value (often 022). But typically, no, I would not want anyone other than myself to write the files that I create. Like, I'm writing code for my awesome new product, and that joker Steve comes along and overwrites all the amazing algorithms with ASCII lolcats and deleted the `.git` directory. Dammit, Steve, you've gone too far!